### PR TITLE
Update web test plan

### PR DESCRIPTION
**Pull request by kimlisa**
- Added a lot of stuff missing
- Re-ordered some tests
- Removed redundant tests
- FYI... there were some `auto` stylistic changes on save from my vscode settings..

Feeling pretty over whelmed by what was missing. Especially the different types of states our web apps has depending on the product type (OSS, team, on-prem, cloud). I think there are still a lot of things missing tbh, like cloud + team only stuff (aws external storage, billing screens, access monitoring), any help will be much appreciated 🙏 

Feel free to add to this PR

### CODE REVIEW

**Review by tigrato — APPROVED** [review]

**Review by mcbattirola — COMMENTED** [review]
Do you think we should add a section for dashboard tenants?

Could be as simple as

- [ ] Can see download page
- [ ] Can download license
- [ ] Can download teleport binaries

**Review by rudream — COMMENTED** [review]
Thanks for this! I've had this in my backlog for the longest time but never had the chance to get to it 😖

File: .github/ISSUE_TEMPLATE/webtestplan.md [hunk 1]
```diff
@@ -1,108 +1,252 @@
...
+Admin refers to users with access_list RBAC defined
+
+- [ ] Renders empty state animation highlighting different features of access lsit
+- [ ] Admins can create, delete, edit, and review access list
+- [ ] Owners (WITHOUT access_list RBAC defined) can still review and edit access lists that they are a owner of
+- [ ] Members can only read access list that they are a member of (edit buttons should be disabled)
+- [ ] Non members, owners, or admins should not be shown any access list
+- [ ] Access list reviews due in two weeks or less
+  - [ ] Red dot is rendered on the notification bell on top bar nav and access lists due are listed
+  - [ ] Red dot is also rendered next to the `Access List` navigation
```
Comment (zmb3) [inline .github/ISSUE_TEMPLATE/webtestplan.md:201]: The red dot tests feel like something that we could do with react testing library and not have to do every quarter, don't you think?

File: .github/ISSUE_TEMPLATE/webtestplan.md [hunk 2]
```diff
@@ -173,6 +318,7 @@ version: v5
 - [ ] Verify you can't review own requests
 
 ### Creating Access Requests (Search Based)
```
Comment (zmb3) [inline .github/ISSUE_TEMPLATE/webtestplan.md:320]:
```suggestion
### Creating Access Requests (Resource Based)
```

While you're here..

File: .github/ISSUE_TEMPLATE/webtestplan.md [hunk 3]
```diff
@@ -1,108 +1,252 @@
...
+    - [ ] Click on all the links and make sure they work (no 404) and are up to date
+    - [ ] Renders cluster information
+    - [ ] OSS: Premium support is locked behind a CTA button
+  - [ ] Can toggle between light and dark theme
+    - [ ] Theme preference is saved upon relogin
 
 ### Unified Resources
-- [ ] Verify that scrolling to the bottom of the page renders more resources
+
+- [ ] Verify that scrolling to - [ ] Verify connecting to a ssh node prompts you to tap your registered WebAuthn keythe bottom of the page renders more resources
```
[the remark this answers is not in the document]
Comment (kimlisa) [reply .github/ISSUE_TEMPLATE/webtestplan.md:60]: i removed the second bullet point

**Comment by kimlisa** [comment]
i copied and pasted this PRs test plan version to our [v16 test plan](https://github.com/gravitational/teleport/issues/42119)

File: .github/ISSUE_TEMPLATE/webtestplan.md [hunk 4]
```diff
@@ -479,412 +624,454 @@ Add the following to enable read access to the auth connectors
...
+    - [ ] Verify that psql shows an appropriate access denied error ("access to db denied. User
+          does not have permissions. Confirm database user and name").
+  - Log in, open a cluster tab, wait for the cert to expire. Switch from a servers view to
+    databases view.
+    - [ ] Verify that a login modal was shown.
+    - [ ] Verify that after logging in, the database list is shown.
+  - Log in, set up two db connections. Wait for the cert to expire. Attempt to connect to the first
+    proxy, then without logging in proceed to connect to the second proxy.
+    - [ ] Verify that an error notification was shown related to another login attempt being in
+          progress.
```
Comment (ravicious) [inline .github/ISSUE_TEMPLATE/webtestplan.md:964]: This seems wrong, each of those main bullet points is duplicated in the Connect test plan. I haven't checked how the points added here differ from those that were already there. But this should be pretty clear once the new points are moved to the correct location.

File: .github/ISSUE_TEMPLATE/webtestplan.md
Comment (ravicious) [inline .github/ISSUE_TEMPLATE/webtestplan.md:1]: It's too late for that, but I'd advise against running autoformatting from one's own editor against md files in the repo. The CI doesn't enforce any particular formatting for md files and we don't know what settings someone uses, so chances are the next person with autoformatting enabled is just going to override the changes anyway. Which overall introduces unnecessary noise in PRs.

I turned off Prettier for md files in my editor for that reason. It's too bad that we can't tell Prettier through its config which files it should format.

**Review by ravicious — APPROVED** [review]

**Review by mcbattirola — APPROVED** [review]

**Review by rudream — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/webtestplan.md
Comment (kimlisa) [reply .github/ISSUE_TEMPLATE/webtestplan.md:1]: ah got it, i'll keep this in mind for mds

**Pull request merged**